react-jsx/react-jsx-quiz-notes.md

```diff
@@ -0,0 +1,36 @@
+# react-jsx-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is JSX?
+it's a syntax extension for JavaScript that lets you write HTML-like code within JavaScript
+- How does React use JSX to render components?
+it makes it so you an have HTML components within a JavaScript code in a declarative way
```

Comment:
JSX Elements are used to make DOM calls that create, modify, and delete DOM elements
A React component returns a JSX Element which represents how the DOM should be structured